src/term/mod.rs

```diff
@@ -1031,6 +1112,32 @@ impl ansi::Handler for Term {
     }
 
     #[inline]
+    fn set_cursor_style(&mut self, mode: ansi::CursorType) {
+        trace!("set_cursor_style: {:?}", mode);
+
+        match mode {
+            ansi::CursorType::BlinkBlock => {
```

Comment:
This function is not really used because I wasn't sure what to put there. Because it should change cursor on something like `echo -e "\e[2 q"` and it should be temporary, it's like a VT100 thing.